aikit/automl/persistence/storage.py

```diff
@@ -0,0 +1,104 @@
+import os
+import json
+import pickle
+import pandas as pd
+from smart_open import open
```

Comment:
maybe encapuslate that within an error catchng and import "regular" open if smart-open is not available